packages/form-js-viewer/package.json

```diff
@@ -8,6 +8,7 @@
       "require": "./dist/index.cjs"
     },
     "./dist/assets/form-js.css": "./dist/assets/form-js.css",
+    "./dist/assets/light.css": "./dist/assets/light.css",
```

Comment:
Let's also rename this to something more meaningful, e.g.

`flatpickr-light.css`

Reply:
I created https://github.com/bpmn-io/form-js/pull/453 to rename the file to something more meaningful. Indeed rollup `rename` option threw some errors, so I moved the stylesheet into `dist/flatpickr` (similar to how we do it with `bpmn-font` in bpmn-js.

Feel free to merge the PR into this one or cherry-pick. You might need to remove the `dist` folders manually in case it doesn't work.